framer/LayerDraggable.coffee

```diff
@@ -570,11 +573,13 @@ class exports.LayerDraggable extends BaseClass
 
 	_stopSimulation: =>
 		@_isAnimating = false
+		@_isMoving = false
+
 		return unless @_simulation
 		@_simulation?.x.stop()
 		@_simulation?.y.stop()
 		@_simulation = null
-		@emit(Events.Move, @layer.point)
+		#@emit(Events.Move, @layer.point)
```

Comment:
The final state is also reached in the animator, the last time it fires will be without movement. Before this change, it fires two move events with no change and no direction, now just one.

But when we fire this one, @isMoving and @isAnimating is already false; all other move events have @isMoving and either @isAnimating or @isDragging. Except perhaps when explicitly calling stop() on an animator.

We can put back this last emit, it is the safer option.